cli/src/cmd/forge/run.rs

```diff
@@ -395,8 +394,8 @@ impl<DB: DatabaseRef> Runner<DB> {
         self.executor.set_balance(address, self.initial_balance);
 
         // Optionally call the `setUp` function
-        Ok(if setup {
-            match self.executor.setup(address) {
+        Ok(match setup_fn {
+            Some(func) => match self.executor.setup(address, &func.name) {
```

Comment:
can we make this an `let res = if let Some(func) setup_func {} else {}`

and do a `Ok(res)` at the end, the `Ok(` across these many lines is rather ugly